packages/suite/src/views/wallet/sign-verify/components/SignMessage/index.tsx

```diff
@@ -0,0 +1,235 @@
+        accountsSelectOpt = accountsSelectArr.map(el => ({
+            label: el.address,
+            value: el.path,
+        }));
+    } else if (account.networkType === 'ethereum') {
+        address = account.descriptor;
+        path = account.path;
+    }
+
+    const { getValues, setValue, errors, formState, reset, control, register } = useForm<Inputs>({
+        defaultValues: {
+            address,
+            path,
+            signMessage: '',
+            accountsSelect: { label: '', value: '', type: '' },
+            signSignature: '',
+        },
+        mode: 'onChange',
+    });
+    const { isValid } = formState;
+
+    useEffect(() => {
+        return () => {
+            reset(); // clear data after coin change
+            path = '';
+            address = '';
+        };
+    }, [reset]);
+
+    return (
+        <Card title={<Translation id="TR_SIGN_MESSAGE" />} isColumn>
+            <Column>
+                <Row>
+                    {account.networkType === 'bitcoin' && (
+                        <Controller
+                            as={<StyledSelect options={accountsSelectOpt} />}
+                            control={control}
+                            topLabel={<Translation id="TR_ADDRESS" />}
+                            name="accountsSelect"
+                            state={errors.accountsSelect ? 'error' : undefined}
+                            onChange={([selected]) => {
+                                path = selected.value;
+                                address = selected.label;
+                                return selected;
+                            }}
+                            bottomText={
+                                errors.accountsSelect && (
+                                    <Translation
+                                        id={(errors.accountsSelect.type as any) as errorsMessage}
+                                    />
+                                )
+                            }
+                            rules={{
+                                validate: {
+                                    TR_REQUIRED_FIELD: (selectValue: Inputs['accountsSelect']) => {
+                                        return selectValue.value.length > 0;
+                                    },
+                                },
+                            }}
+                        />
+                    )}
+                    {/* input is more convenient from the ability to select and copy */}
+                    {account.networkType === 'ethereum' && (
+                        <Input
+                            topLabel={<Translation id="TR_ADDRESS" />}
+                            name="address"
+                            value={address}
+                            type="text"
+                            innerRef={register}
+                            readOnly
+                        />
+                    )}
+                    <StyledIcon
```

Comment:
wrap it into a button with icon, and set `isDisabled` when empty address (move validation from `copyToClipboard`

Reply:
the user does not expect to see a toast message about successful copying, so they can click on the disabled button, I suggest discussing it with the designer